lighthouse-core/test/computed/js-bundles-test.js

```diff
@@ -89,11 +89,10 @@ describe('JsBundles computed artifact', () => {
       Object {
         "files": Object {
           "node_modules/browser-pack/_prelude.js": 480,
-          "null": 104,
```

Comment:
We have a falsy check in our `computeGeneratedFileSizes`, but I think before there was literally a "null" string key there. Might have been a bug that has since been fixed. Anyways, the numbers here still sum up to the same unmapped bytes, so it all looks good.

Reply:
Actually it's probably because I replaced this code to never execute:

https://source.chromium.org/chromium/chromium/src/+/main:third_party/devtools-frontend/src/front_end/core/sdk/SourceMap.ts;l=380;bpv=1

Seems like an ok change. I just didn't want to deal with porting `isRelativeURL`.

Reply:
> I just didn't want to deal with porting isRelativeURL.

wait, isn't that like half the non-type-level changes since last october?

Reply:
I'll look into including ParsedURL, but it's gonna take some refactoring of the build script, I'll do it as a follow up pr.

Reply:
jk, I've updated this PR to include ParsedURL. Those other changes are gone now.